### DISCUSSION TITLE
How do I reset a document you to empty?

### DISCUSSION THREAD

**Discussion by damianthekreeeytor** [bangle-io/bangle.dev#220] [Q&A] [answered]
I would like to reset a document to empty or an initial state. How would I go about doing that or b point in the right direction. I am using the React eco-system.

**Comment by lucywang000** [comment, marked as answer]
You can dispatch a transaction that does something like `tr.replaceWith(0, state.doc.content.size, yourInitialDoc)`

**Reply by damianthekreeeytor** [reply]
Thank you soo much this works great!
